internal/manifests/networkpolicies/operatornetworkpolicies.go

```diff
@@ -0,0 +1,108 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package networkpolicies
```

Comment:
Should we move this into a different path?
e.g. `internal/manifests/operator/networkpolicies.go` or `internal/manifests/manifestutils/networkpolicies.go`?
I expect, `collector`, `opampbridge`, `targetallocator` will all have their own policies?

Reply:
`internal/manifests/manifestutils` - it does not seem like a good fit there

The `internal` package contains a similar functionality in packages:
*  `internal/operator-metrics` 
* `internal/openshift/dashboards` 

I have moved the functionality temporarily to the `intenal/operatornetworkpolicies`. Once this is merged, let's  reorganize it better.